app/assets/javascripts/discourse/app/components/desktop-notification-config.gjs

```diff
@@ -0,0 +1,47 @@
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+import DButton from "discourse/components/d-button";
+import i18n from "discourse-common/helpers/i18n";
+
+export default class DesktopNotificationsConfig extends Component {
+  @service desktopNotifications;
+
+  <template>
+    <div class="controls">
+      {{#if this.desktopNotifications.isNotSupported}}
+        <DButton
+          @icon="bell-slash"
+          @label="user.desktop_notifications.not_supported"
+          @disabled="true"
+          class="btn-default"
+        />
+      {{/if}}
+      {{#if this.desktopNotifications.isDeniedPermission}}
+        <DButton
+          @icon="bell-slash"
+          @label="user.desktop_notifications.perm_denied_btn"
+          @action={{this.recheckPermission}}
```

Comment:
looks like this method got removed?

Reply:
Ahh thank you! That button is always disabled so that action will never be triggered.. not sure what it was doing there to begin with.